src/librustdoc/html/static/main.js

```diff
@@ -1340,6 +1340,17 @@ if (!DOMTokenList.prototype.remove) {
                               "&nbsp;</span></a></td></tr>";
                 });
                 output += "</table>";
+            } else if (location.hostname === "") { // viewing offline
```

Comment:
That won't work if you documented another crate than the std lib...

Reply:
Good point!

What do you think about a condition like `location.hostname === "" && location.href.includes("/doc/rust/html")` instead?

Reply:
That seems over-complicated for something not really worth it. :-/